### ISSUE TITLE
Python 3 support

### ISSUE THREAD

**Issue by kciredor** [fireeye/flare-ida#84]
As you probably already know IDA 7.4 ships with Python 3.x support.
Default may still be Python 2.x, but as of this year Python 2 is deprecated and many plugins are being migrated to 3.x at the moment.

Are you considering supporting Python 3 support?

**Comment by jhsmith** [comment]
Absolutely. We'll be addressing updating these this year. Leaving this issue open as we update.

**Comment by jhsmith** [comment]
Note that we've released some updates for Python3 support in #90 and #92

**Comment by TheDuchy** [comment]
Any updates on idb2pat python3 support? Should I try porting it myself and submitting a pull request?

**Comment by williballenthin** [comment]
@TheDuchy thanks for the offer!

A number of issues have been reported to idb2pat in the past few weeks and coincidentally @mr-tz and I have been spending a lot of time with FLIRT. I think we are likely to re-write the tool in py3 next week. As I have a draft, can I tag you for a review?

To be clear, I don't mean to discourage you if you want to make the port - I just don't want you to do any extra work unnecessarily.

**Comment by TheDuchy** [comment]
Sorry for the late reply, sure thing I'll be glad to help! And yes, if you are doing it I don't see it beneficial for anyone if I did it too.
